tests/unit.py

```diff
@@ -0,0 +1,62 @@
+from PyDexter import PyDexter
+
+def strip_space(str):
+	return ''.join(str.split())
+
+def test_hist():
+	''' indirectly compares actual to expected js generated by PyDexter hist method '''
+	test_input = [1,2,3,4,5]
+	chart_type = 'hist'
+	EXPECTED_JS = strip_space("new D3xter({}).hist([1, 2, 3, 4, 5]); \n")
+	actual_js = strip_space(PyDexter()._make_js(chart_type,test_input))
```

Comment:
Same principle, function arguments get spaces throughout the rest of the lib so it'd be good to do in the tests (there are a few cases).